.xcversion

```diff
@@ -1 +1 @@
-13.4.1
+14.2
```

Comment:
Do we also increase this version each time we update the CI image in the pipelines? I'm not familiar with how we use `.xcversion`, so I wasn't sure.

Reply:
I wasn't sure either 🥲 . But we probably should apply the same changes in [this Woo iOS PR](https://github.com/woocommerce/woocommerce-ios/pull/8033) to this repository.

Reply:
See https://github.com/wordpress-mobile/WordPress-iOS/pull/19834